rpcs3/Emu/Cell/Modules/cellSysutil.cpp

```diff
@@ -212,7 +217,7 @@ error_code cellSysutilGetSystemParamInt(CellSysutilParamId id, vm::ptr<s32> valu
 	break;
 
 	case CELL_SYSUTIL_SYSTEMPARAM_ID_PAD_RUMBLE:
-		*value = CELL_SYSUTIL_PAD_RUMBLE_OFF;
+		*value = CELL_SYSUTIL_PAD_RUMBLE_ON;
```

Comment:
Might as well add a system io setting here

Reply:
It's not going to happen in this pr, I'm just fixing an inconsistency with cellPad here.